src/buffer.cpp

```diff
@@ -124,9 +124,9 @@ CNetBufWithStats::CNetBufWithStats() :
     }
 }
 
-void CNetBufWithStats::GetErrorRates ( CVector<double>& vecErrRates,
-                                       double&          dLimit,
-                                       double&          dMaxUpLimit )
+void CNetBufWithStats::GetErrorRates ( CVector<float>& vecErrRates,
```

Comment:
No need to change to float in this class.

Reply:
Fixed.